3604/1/top_gems.rb

```diff
@@ -0,0 +1,27 @@
+require './array_names_gems_creator.rb'
+require './gem_info_fetcher.rb'
+require './rating_creator.rb'
+require './table_output.rb'
+require './options_creator.rb'
+
+# class for main task: start project
+class TopGems
+  def start_top_gems
```

Comment:
the same as with TableOutput: no need to duplicate class name in method name
please leave just`start`